pages/signup.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://altur-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="../style.css">
+
+    <link rel="stylesheet" href="../stylesheets/signup.css">
+
+    <title>Altur Store | Signup</title>
+</head>
+
+<body>
+
+    <nav class="flex-center-sp-ev">
+        <a href="../index.html">
+            <h1 class="text-xl">Altur Store</h1>
+        </a>
+
+        <div>
+            <input class="text-m search-bar" placeholder="Search" type="text">
+            <i class="search-icon fas fa-search"></i>
+        </div>
+
+        <div class="ecom-icon flex-align-center">
+
+            <div class="links text-m">
+                <a class="log-btn" href="login.html"> Login </a>
+            </div>
+
+            <a href="wishlist.html"><i class="nav-icon wishlist-icon fas fa-heart"></i></a>
+            <div class="top-right badge-style badge-val"> 6 </div>
+
+            <a href="cart.html"> <i class="nav-icon fas fa-shopping-cart"></i></a>
+            <div class="top-right badge-style badge-val"> 2 </div>
+        </div>
+
+    </nav>
+
+    <div class="container flex-center-column">
+
+        <h2 class="text-xl">Sign up</h2>
+        <div class="signup-form flex-center-column">
+            <div>
+                <p>Full Name:</p>
+                <input class="signup-input" type="text" id="name">
+            </div>
+
+            <div>
+                <p>Email: </p>
+                <input class="signup-input" type="text" id="email">
+            </div>
+    
+            <div>
+                <p>Password: </p>
+                <input class="signup-input" type="password" id="password">
+            </div>
+    
+            <div>
+                <p>Confirm Password: </p>
```

Comment:
instead of `<p>` tags, you should use the `<label>` tag to label the inputs

Reply:
Changed tags in the latest commit.